src/main/java/org/arquillian/testcontainers/TestContainersObserver.java

```diff
@@ -22,6 +22,9 @@
 
 @SuppressWarnings("unused")
 class TestContainersObserver {
+
+    private static final String NO_DOCKER_MSG = "No Docker environment is available.";
```

Comment:
```suggestion
    private static final String NO_DOCKER_MSG = "No Docker/podman environment is available.";
```

We are trying to keep this project container runtime-agnostic.